apps/accounts/end_user_signup_forms.py

```diff
@@ -167,7 +170,7 @@ def save(self):
         # TODO: Add Crosswalk Create.
         Crosswalk.objects.create(user=new_user,
                                  user_id_hash=binascii.hexlify(pbkdf2(self.cleaned_data['id_number'],
-                                                                      settings.USER_ID_SALT,
+                                                                      get_user_id_salt(settings.USER_ID_SALT),
```

Comment:
I like this move, I think we should follow the pattern used by oauth2_provider since it's all over the place, so the `settings.USER_ID_SALT` shouldn't be passed in.

Reply:
wups maybe that was for #566, can you fix these prs up so there not on top of eachother or are they dependant?